src/components/UrqlMutation.re

```diff
@@ -39,18 +40,19 @@ let urqlDataToRecord = (result: mutationRenderPropsJs('a)) => {
     | (false, None, None) => NotFound
     };
 
-  {
-    fetching,
-    data,
-    error,
-    executeMutation: result->executeMutationGet,
-    response,
-  };
+  {fetching, data, error, executeMutation, response};
 };
 
 [@react.component]
 let make =
-    (~query: string, ~children: mutationRenderProps('a) => React.element) =>
+    (
+      ~request: UrqlTypes.request('response),
+      ~children: mutationRenderProps('a) => React.element,
```

Comment:
The inferred type here will be `'response`, but I believe it's still good to change `'a` to `'response` to be explicit
```suggestion
      ~children: mutationRenderProps('response) => React.element,
```

Reply:
Ah good catch, thought I got them all!